msteams-platform/resources/design/framework/tabs-mobile.md

```diff
@@ -0,0 +1,113 @@
+
+### Layouts
+
+Choosing the correct layout for your tab is important. You should consider the kind of information you're presenting, and choose a layout that organizes it for easy consumption. Some potential options are outlined below.
+
+#### Single canvas
+
+This is one large area where work gets done. The Wiki app follows this pattern. If you have an app that doesn’t separate content into smaller components this would be a good fit.
+
+![single canvas layout](~/assets/images/mobile-single-canvas.png)
+
+#### List
+
+Lists are great for sorting and filtering large quantities of data and are great at keeping the most important things at the top. It is helpful to use sortable columns. Actions can be added to each list item under the ellipsis menu.
+
+![list layout](~/assets/images/mobile-list.png)
+
+#### Grid
+
+Grids are useful for showing elements which are highly visual. It helps to include a filter or search control at the top.
+
+![grid layout](~/assets/images/mobile-grid.png)
+
+### Tabs with bots on mobile
+
+The below is an example personal app that contains two static tabs and a bot.
+
+![tabs and bots on mobile](~/assets/images/mobile-tab-with-bot.png)
+
+### UI Components
+
+#### Color palettes
+
+Using our approved neutral palette for backgrounds, notifications, text, and buttons will help your app feel more at home in Teams. Since Teams mobile has two colour themes (light and dark), it’s a good idea to make sure your app looks great in both.
+
+##### Light color
+
+![light color palette](~/assets/images/light-color.png)
+
+##### Dark color
+
+![dark color palette](~/assets/images/dark-color.png)
+
+#### Buttons and controls
+
+The way buttons are styled helps communicate what kind of action they trigger. We maintain a wide range of buttons that are formatted to show different levels of emphasis. Buttons can have text, an icon, or a combination of text and an icon. To communicate different levels in a hierarchy, we designed primary and secondary buttons within each category.
+
+![buttons](~/assets/images/buttons.png)
+
+![selection controls](~/assets/images/selection-controls.png)
+
+![chiclets and pills](~/assets/images/chiclets-and-pills.png)
+
+#### Typography
+
+Typography should be clear and purposeful. Emphasize important information and avoid using multiple fonts and sizes to reduce confusion. We recommend using sentence case and avoiding the usage of all caps for localization and legibility.
+
+![mobile typograph](~/assets/images/mobile-typography.png)
+
+#### Fields and Flyouts
+
+Fields are areas where users can input text. Flyouts are more lightweight than dialogs, and appear from the top pane.
+
+##### List controls
+
+![mobile list controls](~/assets/images/mobile-list-controls.png)
+
+##### Field controls
+
+![mobile field controls](~/assets/images/mobile-field-controls.png)
+
+##### Flyout control
```

Comment:
Proposing to remove this particular part :

##### Flyout control
![mobile flyout controls](~/assets/images/mobile-flyout.png)